src/ShaderSwapper.Core/ShaderSwapper.cs

```diff
@@ -30,62 +30,190 @@ public class ShaderSwapper : BaseUnityPlugin
         public const string PluginGUID = "com.deathweasel.bepinex.shaderswapper";
         public const string PluginName = "Shader Swapper";
         public const string PluginNameInternal = Constants.Prefix + "_ShaderSwapper";
-        public const string PluginVersion = "1.6";
+        public const string PluginVersion = "1.7";
```

Comment:
No version bumping in PRs please.
```suggestion
        public const string PluginVersion = "1.6";
```